soc/nxp/imx/imx7d/CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ zephyr_sources(
   soc_clk_freq.c
   )
```

Comment:
s/ressource/resource

Reply:
Have you found anything ? 🙃 
I don't think I have misspelled resource in the sources have I ?